CPAC/utils/build_data_config.py

```diff
@@ -1268,15 +1293,19 @@ def update_data_dct(file_path, file_template, data_dct=None, data_type="anat",
 
         data_dct[site_id][sub_id][ses_id]['brain_mask'] = file_path
 
-    elif data_type == "func":
-
+    elif 'func' in data_type:
         temp_func_dct = {scan_id: {"scan": file_path}}
+        contents = os.listdir(os.path.dirname(file_path))
 
         # scan parameters time
         scan_params = None
         if scan_params_dct:
             scan_params = find_unique_scan_params(scan_params_dct, site_id,
                                                   sub_id, ses_id, scan_id)
+        for file in contents:
+            if '.json' in file:
+                scan_params = os.path.join(os.path.dirname(file_path), file)
+
```

Comment:
Does https://github.com/FCP-INDI/C-PAC/blob/a693b608dde98fe9ea28a6bfab5e365710e5c60d/CPAC/utils/build_data_config.py#L1298-L1307 correctly match up the params with the scans when there're multiple scans?

Reply:
Yes, when there are 2 functional scans, the snac parameters match. This is a screen shot provided by @nathaliaesper when she tested out this branch 
![image](https://github.com/FCP-INDI/C-PAC/assets/58920810/c11fc132-3eba-4f5f-befc-730ee34152ee)

Reply:
Just spoke with Nathalia on some tests she did on this branch, and she noticed that its using one scan parameters for both scans. So I'll fix that and ask for a review again 
![image](https://github.com/FCP-INDI/C-PAC/assets/58920810/3a27b9f1-befa-47ed-bd23-b670262359bf)